client/src/components/User/Suggestions/Suggestions.js

```diff
@@ -0,0 +1,151 @@
+import CreateMonumentSuggestions from '../../Suggestions/CreateMonumentSuggestions/CreateMonumentSuggestions';
+import UpdateMonumentSuggestions from '../../Suggestions/UpdateMonumentSuggestions/UpdateMonumentSuggestions';
+import BulkCreateMonumentSuggestions from '../../Suggestions/BulkCreateMonumentSuggestions/BulkCreateMonumentSuggestions';
+
+/**
+ * Presentational component for displaying a list of Suggestions on the User page
+ */
+export default class Suggestions extends React.Component {
+
+    filterPendingSuggestions() {
+        const { suggestions } = this.props;
+        const pendingSuggestions = {};
+
+        if (suggestions && suggestions.createSuggestions) {
+            pendingSuggestions['create'] = suggestions.createSuggestions.filter(suggestion => !suggestion.isApproved && !suggestion.isRejected);
+        }
+
+        if (suggestions && suggestions.updateSuggestions) {
+            pendingSuggestions['update'] = suggestions.updateSuggestions.filter(suggestion => !suggestion.isApproved && !suggestion.isRejected);
+        }
+
+        if (suggestions && suggestions.bulkCreateSuggestions) {
+            pendingSuggestions['bulk'] = suggestions.bulkCreateSuggestions.filter(suggestion => !suggestion.isApproved && !suggestion.isRejected);
+        }
+
+        return pendingSuggestions;
+    }
+
+    filterApprovedSuggestions() {
+        const { suggestions } = this.props;
+        const approvedSuggestions = {};
+
+        if (suggestions && suggestions.createSuggestions) {
+            approvedSuggestions['create'] = suggestions.createSuggestions.filter(suggestion => suggestion.isApproved);
+        }
+
+        if (suggestions && suggestions.updateSuggestions) {
+            approvedSuggestions['update'] = suggestions.updateSuggestions.filter(suggestion => suggestion.isApproved);
+        }
+
+        if (suggestions && suggestions.bulkCreateSuggestions) {
+            approvedSuggestions['bulk'] = suggestions.bulkCreateSuggestions.filter(suggestion => suggestion.isApproved);
+        }
+
+        return approvedSuggestions;
+    }
+
+    filterRejectedSuggestions() {
+        const { suggestions } = this.props;
+        const rejectedSuggestions = {};
+
+        if (suggestions && suggestions.createSuggestions) {
+            rejectedSuggestions['create'] = suggestions.createSuggestions.filter(suggestion => suggestion.isRejected);
+        }
+
+        if (suggestions && suggestions.updateSuggestions) {
+            rejectedSuggestions['update'] = suggestions.updateSuggestions.filter(suggestion => suggestion.isRejected);
+        }
+
+        if (suggestions && suggestions.bulkCreateSuggestions) {
+            rejectedSuggestions['bulk'] = suggestions.bulkCreateSuggestions.filter(suggestion => suggestion.isRejected);
+        }
+
+        return rejectedSuggestions;
+    }
+
+    renderSuggestions(suggestions, type) {
+        const { role, error } = this.props;
+        const { create, update, bulk } = {...suggestions};
```

Comment:
Any reason for doing `{...suggestions}` instead of `suggestions`? is this like a null check or something?

Reply:
No, no particular reason. I'll change it